docs/documentation/server_admin/topics/clients/client-policies.adoc

```diff
@@ -144,7 +144,7 @@ One of several purposes for this executor is to realize the security requirement
 
 A profile consists of several executors, which can realize a security profile like FAPI and OAuth 2.1. Profile can be configured by the Admin REST API (Admin Console) together with its executors.
 Three _global profiles_ exist and they are configured in {project_name} by default with pre-configured executors compliant with the FAPI 1 Baseline, FAPI 1 Advanced, FAPI CIBA, FAPI 2 and OAuth 2.1 specifications.
-More details exist in the FAPI and OAuth 2.1 section of the link:{adapterguide_link}#_fapi-support[{adapterguide_name}].
+More details exist in the *FAPI* and *OAuth 2.1*  in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section.
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).